included/README.md

```diff
@@ -83,10 +83,15 @@
 [cypress/included:9.3.1](9.3.1) | `cypress/browsers:node16.5.0-chrome94-ff93`
 [cypress/included:9.4.1](9.4.1) | `cypress/browsers:node16.5.0-chrome94-ff93`
 [cypress/included:9.5.0](9.5.0) | `cypress/browsers:node16.5.0-chrome97-ff96`
-[cypress/included:9.5.1](9.5.1) | `cypress/browsers:node16.5.0-chrome97-ff96` 
+[cypress/included:9.5.1](9.5.1) | `cypress/browsers:node16.5.0-chrome97-ff96`
+ 
```

Comment:
nit: extra lines added

Reply:
This is all autogenerated, so I wasn't sure how much I should modify after the fact